components/Onboarding/ConnectViaWallet/ConnectViaWalletTableViewItems.tsx

```diff
@@ -117,8 +116,11 @@ export const InstalledWalletsTableView = memo(
             walletName: wallet.name,
           }),
           action: async () => {
+            const isSCW = !!wallet?.isSmartContractWallet;
             logger.debug(
-              `[Onboarding] Clicked on wallet ${wallet.name} - opening external app`
+              `[Onboarding] Clicked on wallet ${wallet.name} - ${
+                isSCW ? "Opening web page" : "opening external app"
```

Comment:
Can you help me understand the logic of why if we have a smart contract wallet we are opening a webpage and if not, we're opening the external app? Is there no case in which in the future a smart contract wallet will be connected through an external app?

Reply:
currently we don't handle in-app wallets (i.e. smart wallets that Converse controls)
we can either login via an EOA using an external app (Rainbow / Metamask / Coinbase Wallet…)
or login via an external smart wallet (only support right now is Coinbase Smart wallet and it works inside a web modal interface hence the opening web page log)

Reply:
Gotcha! So this may easily change as in external app wallets (or our own wallet handling) evolves to handle SCWs?